docs/index.bs

```diff
@@ -212,6 +212,10 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     A [=/service worker=] has an associated <dfn>list of router rules</dfn> (a [=list=] of {{RouterRule}}s). It is initially an empty [=list=].
 
+    A [=/service worker=] has an associated <dfn>router rule count</dfn> (the number of registered {{RouterRule}}s). It is initially set to zero.
```

Comment:
I guess we do not need a dedicated variable for this because [=list=] has the number of items.
https://infra.spec.whatwg.org/#list-size

Reply:
A stepping back, in spite of the explanation, this variable does not match with the number of rules.  Then, the variable might be needed.
I feel the limitation in this way counterintuitive because the limit does not depend on the number of rules but depends on the number of rules or nests while it is saying limiting with the number of router rules.

Reply:
Discussed offline. We don't need it anymore.

We have a rough agreement on the new approach, having two limits and validates them in between the step 5 and 6 in addRoutes().
https://w3c.github.io/ServiceWorker/#dom-installevent-addroutes